src/components/menu-list/menu-list.scss

```diff
@@ -237,4 +237,3 @@ $list-border-radius: 0.375rem; // 6px
 @import './partial-styles/custom-styles.scss';
 @import './partial-styles/enable-multiline-text.scss';
 @import './partial-styles/_has-grid-layout.scss';
-@import './partial-styles/_static-actions.scss';
```

Comment:
Generally we wanna avoid maintaining duplicate styles. Is there a reason not to import `list/list.scss` into this file, and remove most of the CSS here, keep only menu things?
I don't mean in this PR. But I can look into this as step 4

Reply:
No particular reason other than not having gotten around to even looking at the css yet. I wanted to have a clean split to begin with, since the problem I was trying to address was that limel-menu specific things had crept into limel-list, which isn't good. If we can have a `list.scss` that knows nothing about limel-menu, but is still useful to import in limel-menu, then that's fine with me 🙂

Reply:
Nice. I'll look into that as step 4 when you're ready here.